chisel/src/cli.rs

```diff
@@ -0,0 +1,200 @@
+//! Chisel CLI
+//!
+//! This module contains the core readline loop for the Chisel CLI as well as the
+//! executable's `main` function.
+
```

Comment:
can we move this to `src/bin/chisel.rs` instead? since this is the actual binary

Reply:
Done 👍